elkserver/docker/redelk-base/redelkinstalldata/scripts/modules/alarm_useragent/module.py

```diff
@@ -65,7 +66,7 @@ def alarm_check(self):
                 qSub = qSub + " OR http.headers.useragent:%s" % keyword
         qSub = qSub + ") "
         #q = "%s AND redir.backendname:c2* AND tags:enrich_* AND NOT tags:alarm_* "%qSub
-        q = "%s AND redir.backend.name:c2* AND NOT tags:%s"%(qSub,info['submodule'])
+        q = "%s AND redir.backend.name:c2* AND NOT tags:alarm_useragent" % qSub
```

Comment:
Use the `submodule` name instead of hardcoding `alarm_useragent`

Reply:
true, likely has to be self.info as mentioned in some other item.